cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml

```diff
@@ -0,0 +1,36 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
+type: automl
+
+experiment_name: dpv2-cli-automl-forecasting-github-dau-experiment
+description: A Time-Series Forecasting job using Github DAU dataset that trains only the TCNForecaster model.
+
+compute: azureml:automl-gpu-cluster
+
+task: forecasting
+primary_metric: normalized_root_mean_squared_error
+
+log_verbosity: info
+
+target_column_name: "count"
+training_data:
+  path: "./training-mltable-folder"
+  type: mltable
+validation_data:
+  path: "./validation-mltable-folder"
+  type: mltable
+
+limits:
+  timeout_minutes: 60
+  max_concurrent_trials: 4
+  enable_early_termination: true
+
+forecasting:
+  time_column_name: "date"
+  forecast_horizon: 14
+  frequency: "D"
```

Comment:
We can remove frequency here for parity with the notebook.

Reply:
Thanks! I have removed frequency from the config file.